course_content/iris_course/7.Advanced_Concepts.ipynb

```diff
@@ -494,7 +494,7 @@
    "cell_type": "markdown",
    "metadata": {},
    "source": [
-    "As you can see, by loading a "
+    "As you can see, by loading a cube before applying the constraints and extractions it will take significantly longer when compared to applying the same constraints and extraction operations during the loading process."
```

Comment:
You might be surprised to learn that this isn't true! Firstly remember that *loading* is just bringing the structure and metadata into memory - the data itself remains lazy, making it less expensive to load the 'whole' `Cube`.

With NetCDF files the performance of constrain-first vs load-first _should_ be equivalent, but for PP files it is much faster to load-first. This is because a `Constraint` is applied to each PP field in turn during loading, making it faster to load everything at once and only apply the `Constraint` afterwards.

I don't know what exactly this markdown cell was aiming to show. At a bare minimum it shows two different approaches that can be used, each of which might be a better fit for different workflows.